config/locales/en.yml

```diff
@@ -27,3 +27,9 @@ en:
 
   titles:
     application: Budgetapp
+
+  activerecord:
+    errors:
+      messages:
+        blank: "can't be blank"
+        taken: "already taken"
```

Comment:
This change is also in another PR. A suggestion to rebase this once the other PR is merged (or vice versa)